utils/node_local_test.py

```diff
@@ -2521,9 +2521,11 @@ def _post(self, rc):
             # It's not possible to log these as warnings, because there is
             # no src line to log them against, so simply assert.
             assert self.returncode == 0
-            assert self.stderr == b''
-            if self.aft.expected_stdout is not None:
-                assert self.stdout == self.aft.expected_stdout
+
+            if self.check_post_stdout:
```

Comment:
(pylint-no-member) Instance of 'AllocFailTestRun' has no 'check_post_stdout' member